envelopes/lib/envelopes/current_user.rb

```diff
@@ -3,8 +3,10 @@ class CurrentUser
         USER_FILES_PATH = "#{__dir__}/../../users"
         CURRENT_USER = "#{USER_FILES_PATH}/current_user.yaml"
         
-        def initialize
-            @current_user = YAML.load(File.read(CURRENT_USER))
+
+        def self.current_user
+            #should I assign current user to a variable?
```

Comment:
No it looks like you can just return the value in the file.
I mean you don't even need to use YAML  it looks like, just `File.read(CURRENT_USER)`.  It's just a single line of text it seems like.

Reply:
👍